src/renderer/modals/VoteTron/fields/VotesField.js

```diff
@@ -156,38 +183,55 @@ type SRRowProps = {
   isSR?: boolean,
   value?: number,
   disabled?: boolean,
-  onChange: (vote: number) => void,
+  max: number,
+  onChange: (address: string, vote: number) => void,
   onExternalLink: (address: string) => void,
 };
 
-const SRRow = ({ sr, rank, isSR, value, disabled, onChange, onExternalLink }: SRRowProps) => (
-  <Row disabled={value && disabled} active={!!value}>
-    <IconContainer isSR={isSR}>{isSR ? <Trophy size={16} /> : <Medal size={16} />}</IconContainer>
-    <InfoContainer>
-      <Title onClick={() => onExternalLink(sr.address)}>
-        <Text>{sr.name}</Text>
-        <IconContainer>
-          <ExternalLink size={16} />
+const SRRow = ({ sr, rank, isSR, value, disabled, max, onChange, onExternalLink }: SRRowProps) => {
```

Comment:
making this a React.memo should help perf (assuming the props are all protected by ref changes, like with useCallback,..)